src/render/meshes/base.js

```diff
@@ -78,23 +78,19 @@ export class Base extends Renderable {
   _material(options) {
     const precision = this.renderer.capabilities.precision;
 
-    const vertexPrefix = `\
-    precision ${precision} float;
```

Comment:
I sort of suspect that a leading newline here would be irrelevant. I also suspect that the 4 spaces before "precision" are irrelevant. But it's not really something I can speak with authority on, and I can't find a good reference for it 😐